src/components/pages/Authentication/index.tsx

```diff
@@ -1,48 +1,42 @@
 import * as React from 'react';
 import styled from '@emotion/styled';
 import { connect } from 'react-redux';
+
 import AuthenticationLayout from 'components/layouts/AuthenticationLayout';
 import __spacing from 'settings/__spacing';
 import { fontSizes } from 'settings/__fonts';
 import { GRAY, BLACK, GAINS_BORO } from 'settings/__color';
 import Login from './Login';
 import SignUp from './SignUp';
 
+interface AuthenticationProps {
```

Comment:
I think we might want to use a different name for this variable here as that name had already occurred in other files.

Reply:
See comment on renaming this above ☝️